clients/vscode/src/index.ts

```diff
@@ -31,13 +38,13 @@ export function activate(context: ExtensionContext): void {
 
   // Options to control the language client
   const clientOptions: LanguageClientOptions = {
-    // Register the server for plain text documents
+    // Register the server for marko text documents
```

Comment:
[marko](https://markojs.com/)?

Reply:
yep,  I changed that, sohuld be prisma :D But the marko language server had a nice approach to the language server so that's why I did it similar here